src/config/effect/delay.rs

```diff
@@ -21,6 +17,7 @@ pub struct DelayConfig {
     #[serde(default = "DelayConfig::default_num")]
     pub num: u32,
     pub delay_ms_slider: Option<MidiSlider>,
+    pub tap_tempo: Option<NoteOn>,
 }
 
 impl DelayConfig {
```

Comment:
I split the delay config out into its own file since it was getting kind of large.